monkey/tests/unit_tests/infection_monkey/ransomware/test_utils.py

```diff
@@ -0,0 +1,91 @@
+import os
+
+import infection_monkey.ransomware.utils
+
+VALID_FILE_EXTENSION_1 = "file.3ds"
+VALID_FILE_EXTENSION_2 = "file.zip"
+INVALID_FILE_EXTENSION_1 = "file.jpe"
+INVALID_FILE_EXTENSION_2 = "file.xyz"
+SUBDIR_1 = "subdir1"
+SUBDIR_2 = "subdir2"
```

Comment:
Where's a test for a file like `file.jpg.zip`? Based on the if statement, I'd predict that it would fail.

Reply:
It passes, but I'll add a file like that.